lib/project_types/script/layers/infrastructure/assemblyscript_project_creator.rb

```diff
@@ -56,7 +56,7 @@ def write_package_json
               },
               "scripts": {
                 "test": "asp --summary --verbose",
-                "build": "shopify-scripts-toolchain-as build --src src/script.ts --binary #{script_name}.wasm -- --lib node_modules --optimize --use Date="
+                "build": "shopify-scripts-toolchain-as build --src src/script.ts --binary build/#{script_name}.wasm -- --lib node_modules --optimize --use Date="
```

Comment:
To doublecheck, will this create the build directory if it doesn't already exist?

Reply:
Yes.